pkgs/tools/misc/zoxide/default.nix

```diff
@@ -15,15 +16,17 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "zoxide";
-  version = "0.5.0";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "ajeetdsouza";
     repo = "zoxide";
     rev = "v${version}";
-    sha256 = "143lh94mw31pm9q7ib63h2k842g3h222mdabhf25hpb19lka2w5y";
+    sha256 = "sha256-ZeGFsVBpEhKi4EIhpQlCuriFzmHAgLYw3qE/zqfyqgU=";
   };
 
+  buildInputs = lib.optionals stdenv.isDarwin [ libiconv ];
```

Comment:
Just want to mention I know about `lib.optional` but I prefer `lib.optionals` here as it is more extensible

Reply:
lib.optionals is preferred in my opinion if you are working with lists.